phpcs/Custom/Sniffs/Classes/ClassLinesOfCodeSniff.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Ensures that classes do not exceed a certain amount of lines of code.
+ */
+class Custom_Sniffs_Classes_ClassLinesOfCodeSniff implements PHP_CodeSniffer_Sniff
+{
+    /**
+     * The limit of lines of code a class should not exceed.
+     *
+     * @var int
+     */
+    public $linesLimit = 400;
```

Comment:
I would prefer calling it `$softLinesLimit` instead or `$warningLinesLimit` (and `$errorLinesLimit`) to make it more self-explaining.

Reply:
I used the naming convention from the official sniffs: https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Standards/Generic/Sniffs/Files/LineLengthSniff.php#L28